package.json

```diff
@@ -0,0 +1,268 @@
+        "email": "ryanlanese@gmail.com"
+      },
+      {
+        "name": "Kevin Lawler",
+        "email": "kevin@brave.com"
+      },
+      {
+        "name": "Bartlomiej Lechowski",
+        "email": "btlechowski@gmail.com"
+      },
+      {
+        "name": "Brian Lee"
+      },
+      {
+        "name": "Ailin Liao"
+      },
+      {
+        "name": "Jocelyn Liu",
+        "email": "yrliou@gmail.com"
+      },
+      {
+        "name": "Karen Liu",
+        "email": "kliu@brave.com"
+      },
+      {
+        "name": "Tom Lowenthal",
+        "email": "me@tomlowenthal.com"
+      },
+      {
+        "name": "Terry Mancey",
+        "email": "tmancey@brave.com"
+      },
+      {
+        "name": "Pete Miller",
+        "email": "miller.pete@gmail.com"
+      },
+      {
+        "name": "Ross Moody",
+        "email": "fiftyfivehis@gmail.com"
+      },
+      {
+        "name": "Mark Pilgrim",
+        "email": "mpilgrim@brave.com"
+      },
+      {
+        "name": "Mihai Plesa",
+        "email": "mihai.plesa@gmail.com"
+      },
+      {
+        "name": "Brad Richter",
+        "email": "bradley.richter@gmail.com"
+      },
+      {
+        "name": "Jenn Rhim",
+        "email": "jennrhim@gmail.com"
+      },
+      {
+        "name": "Riastradh"
+      },
+      {
+        "name": "Garrett Robinson",
+        "email": "garrett.f.robinson@gmail.com"
+      },
+      {
+        "name": "Emerick Rogul",
+        "email": "erogul@brave.com"
+      },
+      {
+        "name": "Jason Sadler",
+        "email": "jsadler@brave.com"
+      },
+      {
+        "name": "Geeta Sarvadnya",
+        "email": "gsarvadnya@brave.com"
+      },
+      {
+        "name": "Mandar Shinde",
+        "email": "mandar@brave.com"
+      },
+      {
+        "name": "Peter Snyder",
+        "email": "psnyder@brave.com"
+      },
+      {
+        "name": "David Temkin",
+        "email": "temkin@gmail.com"
+      },
+      {
+        "name": "Anthony Tseng",
+        "email": "darkdh@gmail.com"
+      },
+      {
+        "name": "Sriram Bellur Venkataram",
+        "email": "srirambv@gmail.com"
+      },
+      {
+        "name": "Lauren Wagner",
+        "email": "lauren@brave.com"
+      },
+      {
+        "name": "Ryan Watson",
+        "email": "w0ts0n@gmail.com"
+      },
+      {
+        "name": "Nejc Zdovc",
+        "email": "nejc@brave.com"
+      },
+      {
+        "name": "Yan Zhu",
+        "email": "yan@mit.edu"
+      },
+      {
+        "name": "Sergey Zhukovsky",
+        "email": "serg@brave.com"
+      }
+    ],
+    "license": "MPL-2.0",
+    "bugs": {
+      "url": "https://github.com/brave/brave-browser/issues"
+    },
+    "homepage": "https://github.com/brave/brave-browser#readme",
+    "dependencies": {
+      "chalk": "^2.4.2",
+      "commander": "^2.9.0",
+      "fs-extra": "^8.0.1",
+      "log-update": "^3.2.0"
+    },
+    "devDependencies": {
+      "ip": "^1.1.5",
+      "jest": "^24.8.0"
+    }
+  }
```

Comment:
Maybe add EOL here to be consistent with other package.json

Reply:
done